### PR TITLE
adds boiler plate code for velocity graph

### PR DESCRIPTION
**Pull request by AshankDsouza**

### CODE REVIEW

File: client/components/velocityGraph.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  SelectLabel,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select"
+import { useEffect, useState } from "react"
+import BarGraph from "./barGraph"
+
+
+function VelocityGraph(props: any) {
+  const {slug} = props;
```
Comment (SukanyaRani) [inline client/components/velocityGraph.tsx]: Here slug means the slugName or the id?

**Review by SukanyaRani — CHANGES_REQUESTED** [review]
Use

File: client/components/velocityGraph.tsx (hunk 1 above)
Comment (AshankDsouza) [reply client/components/velocityGraph.tsx]: Done.

**Review by SukanyaRani — APPROVED** [review]
Looks good to me

**Pull request merged**